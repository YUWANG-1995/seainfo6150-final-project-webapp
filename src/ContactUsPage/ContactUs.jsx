import React from "react";
import styles from "./ContactUs.module.css";

const ContactUs = () => {
  return (
    <div className={styles.container}>
      <h1>CONTACT US</h1>
      <div>
        <address>
             401 Terry Ave N #103<br/>
             Seattle, WA 98109<br/>
             USA<br/>
            <a href="mailto:wang.yu14@northeastern.edu">Custoer Service Email</a><br/>
            Call us : +1 (877)-668-7727<br/>
        </address>
      </div>
    </div>
  )
}

export default ContactUs
