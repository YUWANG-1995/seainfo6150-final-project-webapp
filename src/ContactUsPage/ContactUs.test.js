import React from "react";
import { render } from "@testing-library/react";
import ContactUs from "./ContactUs.jsx";

it("renders the ContactUs Component Correctly", () => {
  const { container } = render(
      <ContactUs />
  );
  expect(container).toMatchSnapshot();
});
