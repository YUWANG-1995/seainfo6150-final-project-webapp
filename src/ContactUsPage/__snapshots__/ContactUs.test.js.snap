// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ContactUs Component Correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      CONTACT US
    </h1>
    <div>
      <address>
        401 Terry Ave N #103
        <br />
        Seattle, WA 98109
        <br />
        USA
        <br />
        <a
          href="mailto:wang.yu14@northeastern.edu"
        >
          Custoer Service Email
        </a>
        <br />
        Call us : +1 (877)-668-7727
        <br />
      </address>
    </div>
  </div>
</div>
`;
